SYCL/ESIMD/histogram_256_slm.cpp

```diff
@@ -63,9 +63,11 @@ ESIMD_INLINE void histogram_atomic(const uint32_t *input_ptr, uint32_t *output,
   simd<uint, 16> local_histogram;
   local_histogram = slm_gather<uint32_t, 16>(slm_offset);
   atomic_update<atomic_op::add, uint32_t, 8>(
-      output, slm_offset.select<8, 1>(0), local_histogram.select<8, 1>(0), 1);
+      output, simd<uint, 8>(slm_offset.select<8, 1>(0)),
```

Comment:
minor comment. This can be written just a bit simpler:
```suggestion
      slm_offset.select<8, 1>(0).read(),
```